internal/gatewayapi/validate.go

```diff
@@ -303,7 +304,7 @@ func (t *Translator) validateTerminateModeAndGetTLSSecrets(listener *ListenerCon
 			break
 		}
 
-		err := validateTLSSecretData(secret)
+		err := validateTLSSecretsData(secrets, listener.Hostname)
```

Comment:
is this always a non `nil` / empty value ?

Reply:
I think computeHosts got covered all of the cases, so it will default to [*]